src/core/Akka/Util/Internal/TaskExtensions.cs

```diff
@@ -20,40 +20,6 @@ namespace Akka.Util.Internal
     [InternalApi]
     public static class TaskExtensions
     {
-        /// <summary>
-        /// TBD
-        /// </summary>
-        /// <typeparam name="TTask">TBD</typeparam>
-        /// <typeparam name="TResult">TBD</typeparam>
-        /// <param name="task">TBD</param>
-        /// <returns>TBD</returns>
-        public static Task<TResult> CastTask<TTask, TResult>(this Task<TTask> task)
```

Comment:
Let's not use this guy anymore, but the features of the C# itself.

Reply:
Agree